Makefile

```
@@ -1,9 +1,9 @@
 
 # Image URL to use all building/pushing controller image targets
-CONTROLLER_IMG ?= docker.io/anik120/catalogsource-controller:latest
+CONTROLLER_IMG ?= quay.io/operator-framework/catalogd-controller:latest
 
 # Image URL to use all building/pushing apiserver image targets
-SERVER_IMG ?= docker.io/anik120/rukpak-packageserver:latest
+SERVER_IMG ?= quay.io/operator-framework/catalogd:latest
```

Nit: Should we have the custom apiserver be called `catalogd-server`? IMO `catalogd` is just the umbrella name for when both the `catalogd-controller` and `catalogd-server` are deployed on cluster.
```suggestion
SERVER_IMG ?= quay.io/operator-framework/catalogd-server:latest
```

---

That's a good point. Renamed this to `catalogd-server`